src/PrestaShopBundle/Form/Admin/Sell/Customer/CustomerType.php

```diff
@@ -306,6 +308,22 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     ),
                     'required' => false,
                     'invalid_message' => $this->trans('This field is invalid.', 'Admin.Notifications.Error'),
+                    'constraints' => [
+                        new GreaterThanOrEqual([
+                            'value' => 0,
+                            'message' => $this->trans(
+                                '%s is invalid. Please enter an integer greater than or equal to 0.',
+                                'Admin.Notifications.Error'
+                            ),
+                        ]),
+                        new LessThan([
+                            'value' => 4294967296,
+                            'message' => $this->trans(
+                                '%s is invalid. Please enter an integer lower than %s.',
+                                'Admin.Notifications.Error'
+                            ),
+                        ]),
```

Comment:
Shorter, faster, easier ?
```suggestion
                        new Range([
                            'min' => 0,
                            'max' => 4294967295,
                            'minMessage' => $this->trans(
                                '%s is invalid. Please enter an integer greater than or equal to 0.',
                                'Admin.Notifications.Error'
                            ),
                            'maxMessage' => $this->trans(
                                '%s is invalid. Please enter an integer lower than %s.',
                                'Admin.Notifications.Error'
                            ),
                        ]),
```

Reply:
I didn't know it. Thanks